melodic/distribution.yaml

```diff
@@ -6802,6 +6802,22 @@ repositories:
       url: https://github.com/nobleo/mobile_robot_simulator.git
       version: master
     status: maintained
+  mocap_nokov:
+    doc:
+      type: git
+      url: https://github.com/NOKOV-MOCAP/mocap_nokov.git
+      version: master
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/NOKOV-MOCAP/mocap_nokov_release.git
+      version: 0.0.2-2
+    source:
+      test_pull_requests: true
```

Comment:
@duguguang has this been [setup](http://wiki.ros.org/buildfarm/Pull%20request%20testing) accordingly?

Reply:
yes